tests/samplers_tests/tpe_tests/test_parzen_estimator.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from optuna.samplers.tpe.parzen_estimator import ParzenEstimator
+from optuna.samplers.tpe.sampler import default_weights
+
+
+class TestParzenEstimator(object):
```

Comment:
As there're many duplicated logics b/w the two test cases, could you extract them to a common one? (E.g., with `pytest.mark.parametrize`.)

Reply:
Thank you for your suggestion. As you mentioned, I used `pytest.mark.parametrize` to remove the duplication. Any feedback is welcome.